db/migrate/20210610150945_create_deltas.rb

```diff
@@ -0,0 +1,13 @@
+Sequel.migration do
+  change do
+    create_table :deltas do
```

Comment:
I don't think this table name actually describes what's in this table. So technically these are change_events that get created when a certain trigger happens - in our case when we detect that a certain difference occurs between two versions of the same object. Not sure if we can come up with a different name that would flag to a developer more automatically what these are all about?

Reply:
The name is technically correct (the "delta", aka changes in the data, between one day and the previous) but not very descriptive.
I'll think about it while I fix the rest